tests/foreman/ui/test_contentview.py

```diff
@@ -912,11 +912,61 @@ def test_positive_promote_with_rh_custom_spin(self):
 
         @assert: Content view can be promoted
 
-        @caseautomation: notautomated
-
-
         @CaseLevel: System
         """
+        cv_name = gen_string('alpha')
+        filter_name = gen_string('alpha')
+        rh_repo = {
+            'name': REPOS['rhst7']['name'],
+            'product': PRDS['rhel'],
+            'reposet': REPOSET['rhst7'],
+            'basearch': 'x86_64',
+            'releasever': None,
+        }
+        env_name = gen_string('alpha')
+        strategy, value = locators['content_env.select_name']
+        # Create new org to import manifest
+        org = entities.Organization().create()
+        with Session(self.browser) as session:
+            make_lifecycle_environment(
+                session, org=org.name, name=env_name)
+
+            self.assertIsNotNone(session.nav.wait_until_element(
+                (strategy, value % env_name)))
+
+            self.setup_to_create_cv(rh_repo=rh_repo, org_id=org.id)
+            # Create content-view
+            make_contentview(session, org=org.name, name=cv_name)
+            self.assertIsNotNone(self.content_views.search(cv_name))
+            # Add rh repo
+            self.content_views.add_remove_repos(cv_name, [rh_repo['name']])
+            self.assertIsNotNone(self.content_views.wait_until_element(
+                common_locators['alert.success_sub_form']))
+
+            # add a filter
+            self.content_views.add_filter(
+                cv_name,
+                filter_name,
+                FILTER_CONTENT_TYPE['package'],
+                FILTER_TYPE['exclude'],
+            )
+            # assert the added filter visible
+            self.assertIsNotNone(self.content_views.search_filter(cv_name,
+                                                                  filter_name))
+            # exclude some package in the created filter
+            self.content_views.add_packages_to_filter(cv_name, filter_name,
+                                                      ['gofer'],
```

Comment:
or we can totally remove that line at all,
but logically if we make a filter it should contain something to filter (some thing to include or to exclude).
this package is used for the moment only for these two tests.
Perhaps to not make a constant we can create an errata filter instead, what do you think?